frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,23 +1,66 @@
-import React from "react";
-import { useAuth } from "../../services/auth";
-import "./dashboard.module.css";
-import Sidebar from "../../components/Sidebar/Sidebar";
-import sbStyle from "../../components/Sidebar/sidebar.module.css";
+import React, { useState } from 'react';
+import { useAuth } from '../../services/auth';
+import styles from './dashboard.module.css';
+import Sidebar from '../../components/Sidebar/Sidebar';
+import sbStyles from '../../components/Sidebar/sidebar.module.css';
+import Logo from "../../assets/quiz_qrafter_logo_dark.svg";
+import { useTheme } from '../../containers/Admin/ThemeContext';
+import Quiz from '../../containers/Admin/Quiz';
+import Settings from '../../containers/Admin/Settings';
 
 const Dashboard: React.FC = () => {
-  const { signOut } = useAuth();
+  const { user, signOut } = useAuth();
+  const themeColors = {
+    light: '#7a84f9',
+    dark: '#808080'
+  };
+
+  const { theme} = useTheme();
+  const [sidebarOpen, setSidebarOpen] = useState(false);
+  const currentColor = themeColors[theme];
+  const [activeMenu, setActiveMenu] = useState<'quiz' | 'settings'>('quiz');
 
   const handleSignOut = () => {
     signOut();
   };
 
+  const toggleSidebar = () => {
+    setSidebarOpen(!sidebarOpen);
+  };
+
   return (
-    <div>
-      {/* <h1>Dashboard</h1>
-      <button onClick={handleSignOut}>Logout</button> */}
-      <div className = {sbStyle.sidebar}><Sidebar /></div>
-      {/* Add your dashboard content here */}
-    </div>
+    <>
+      <div className={styles.dashboard}>
+        <header className={styles.header} style={{ backgroundColor: currentColor }}>
+          <div className={styles.logoMenu}>
+            <button onClick={toggleSidebar} className={styles.menuButton}>
+              ☰
+            </button>
+
+          </div>
+          <img src={Logo} alt="Quiz Qrafter" className={styles.dashboardLogo} />
+          <div className={styles.headerContent}>
+            {/* <span className={styles.email}>{user?.email}</span> */}
```

Comment:
I didn't forget the commented line. It's commented because the service has not been created for it, so I thought to show a conceptual version of it. It can be uncommented when the endpoint is ready so that it actually shows the signed in user email.

Also, I would need more description of what you described for the first part. Message me on Discord.